### DISCUSSION TITLE
Sometimes WebUI does not respond when running in cloud and using share option

### DISCUSSION THREAD

**Discussion by ShadeRedeye** [vladmandic/sdnext#212] [General]
### Issue Description

i think it's become common problem when use it in cloud service such paperspace or colab .and i didn't know what cause it.

sometimes when generate image it didn't want to process it,(i didn't see it progressbar in console,so i think it didn't run)
also whenever i click below "generate" button,after change theme and reload it via "Restart UI" button,WebUI become hang.

**Comment by vladmandic** [comment]
i just tried and cannot reproduce:
1. click generate
2. go to settings -> user interface -> ui theme
3. change theme
4. press apply settings
5. press restart ui
6. wait for ui reload
7. click generate

if you're having an issue, i need a clear reproducible steps otherwise not much i can do.

**Comment by ShadeRedeye** [comment]
hmm seems like local ver didn't have problem like that,maybe tryin via cloud ver,and add --share args,if that still didn't happen,maybe my laptop can't handle papespace website. :(

i use paperspace,and these notebook to run a webui but with minor change,such use your repo
https://github.com/Engineer-of-Stuff/stable-diffusion-paperspace

**Comment by vladmandic** [comment]
more likely a connection reset between your home and paperspace.
i've just added `--disable-queue` to cmd flags that disables Gradio queues and forces it to use HTTP instead of WebSockets, can you try?

but if you're running webui behind notebook and using share, that's beyond me - that fits in discussions, not an active issue, converting - maybe someone there has more experience with that.

**Comment by ShadeRedeye** [comment]
just,try it,it's more better now it,even faster,for some webui bug i discover when use 
--disable-queue, i got message that relate xdg-utils and something like "< html >" and it looks like causing image result didn't show up after generate

for rest better waiting a guys that have similliar problem like me completed his test,btw thank for your great work.i really appreciated it :)
